### DISCUSSION TITLE
How to share MIMIC-IV data structure with someone without access rights?

### DISCUSSION THREAD

**Discussion by LevanBokeria** [MIT-LCP/mimic-code#1816] [MIMIC-IV] [answered]
Hello,

Some collaborators have asked us if they could see a data sample from MIMIC-IV which would give them an idea of the data structure. 

Is there any such publically available data sample out there being fully anonymised but retaining the structure, such as field names, dimensions, etc etc. 

If such an example does not exist, would it be permissible if those of us with MIMIC-IV access rights subsample the dataset, remove all data and replace them with random numbers or strings, and share that resulting dataset? It would have the structure of MIMIC-IV without any original data being there. 

Thank you!

**Comment by LevanBokeria** [comment]
I've found out there is the MIMIC-III Demo dataset: https://physionet.org/content/mimiciii-demo/1.4/

Is there one for MIMIC-IV?

**Comment by heisenbug-1** [comment, marked as answer]
Hey, MIMIC-IV also has a demo dataset (I think it contains data for 100 patients): https://physionet.org/content/mimic-iv-demo/2.2/

**Reply by LevanBokeria** [reply]
Thanks!
